src/template/minimal/_site/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
Er these files are not supposed to be in the commit right? The generated sites shouldn't be in the template.